lua/lspsaga/hover.lua

```diff
@@ -211,14 +211,38 @@ end
 
 function hover:do_request(args)
   local params = util.make_position_params()
+
+  local expected_result_count = 0
+  lsp.for_each_buffer_client(0, function(client)
```

Comment:
does there can use `libs.get_clinet_by_cap` by using `hoverProvider`

Reply:
The issue with that is that it would only return the first one that *could* return a valid hover result. In the case of the two servers, if you hover over a string with a tailwindcss class in it, tailwindcss server will return a hover and tsserver won't, but otherwise in most other cases only tsserver will return a hover result. If I used get_client_by_cap, it would choose one of the two for all supported filetypes right?